pkg/bmc/bmc.go

```diff
@@ -472,6 +472,40 @@ func (bmc *BMC) SystemPowerCycle() error {
 	return bmc.SystemResetAction(redfish.PowerCycleResetType)
 }
 
+// SystemPowerState returns the system's current power state using the Redfish API.
+// Returned string can be one of On/Off/Paused/PoweringOn/PoweringOff.
+func (bmc *BMC) SystemPowerState() (string, error) {
+	if valid, err := bmc.validateRedfish(); !valid {
+		return "", err
+	}
+
+	glog.V(100).Info("Collecting current power state from bmc's redfish endpoint")
+
+	redfishClient, cancel, err := redfishConnect(
+		bmc.host,
+		bmc.redfishUser.Name, bmc.redfishUser.Password,
```

Comment:
Please move bmc.redfishUser.Password, to the next line according to https://github.com/openshift-kni/eco-goinfra#functions-format

Reply:
Done in https://github.com/openshift-kni/eco-goinfra/pull/442/commits/7c150595b63e7b41722bfc565b1898a00020db4b